crates/infra/utils/src/errors/mod.rs

```diff
@@ -4,14 +4,15 @@ use crate::paths::PathExtensions;
 use anyhow::{bail, Result};
 use ariadne::{Color, Label, Report, ReportKind, Source};
 
-#[derive(Debug)]
+#[allow(clippy::len_without_is_empty)]
+#[derive(Debug, Default)]
 pub struct InfraErrors {
     contents: Vec<ErrorDescriptor>,
 }
 
 impl InfraErrors {
     pub fn new() -> Self {
-        return Self { contents: vec![] };
+        return Self::default();
```

Comment:
I find it clearer to impl `new` myself and explicitly implement `Default` using the `new` but in this case it's a struct with only a single `Vec` field that's defined few lines above, so I wanted to avoid the boilerplate of the manual `Default` impl here